CHANGELOG.md

```diff
@@ -1,18 +1,31 @@
 # Changelog
 
-## [Unreleased]
+## [1.1.0] - 2019-6-14
 
-### Added
+## Added
```

Comment:
looks like we're using three hashes for these items below (so they're smaller than the version numbers)

Reply:
I've fixed this and the the [Unreleased] marker.